packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -94,13 +108,21 @@ export async function stripeCheckout({
         },
       })
 
-      const merchantCoupon = couponId
-        ? await getMerchantCoupon({
-            where: {
-              id: couponId as string,
-            },
-          })
-        : false
+      let merchantCoupon = await getMerchantCoupon({
+        where: {
+          id: couponId,
+        },
+      })
+
+      // Null out the `merchantCoupon` if it is a PPP coupon when the
+      // quantity is greater than 1. You cannot apply PPP to a purchase
+      // of multiple seats.
+      if (merchantCoupon?.type === 'ppp' && quantity > 1) {
+        // TODO: Let support know that we may have showed a discounted
+        // price to a user when we didn't mean to.
+
+        merchantCoupon = null
+      }
```

Comment:
Here is where the coupon logic is handled - it basically clears the `merchantCoupon` if it is PPP and the quantity is greater than 1.

I'm also thinking that if this service is handling a scenario where we have a PPP coupon and a multi-seat purchase, then it is possible one of the client apps has a bug and that we displayed the wrong price to a user. In that case, we should notify support so that they are aware this is happening. Would that make sense as some kind of `Sentry` call?